src/Kunstmaan/PagePartBundle/Event/PagePartEvent.php

```diff
@@ -42,4 +46,14 @@ public function setResponse(Response $response)
     {
         $this->response = $response;
     }
+
+    public function getPage(): ?HasPagePartsInterface
+    {
+        return $this->page;
+    }
+
+    public function setPage(?HasPagePartsInterface $page): void
```

Comment:
Do you specifically need the setter? Otherwise I would propose to leave it out, we can always add afterwards if there is a use case

Reply:
Not needed so I removed this setter